data/utils/defectsFromBias.py

```diff
@@ -21,86 +21,20 @@
 # see <http://www.lsstcorp.org/LegalNotices/>.
 #
 import argparse
-import time
 
-import numpy
-from astropy.io import fits
+from dateutil import parser as date_parser
```

Comment:
I suggest `import dateutil`...`dateutil.parser`. It's only a bit shorter and avoids a confusing rename.

Reply:
If I don't import parser from dateutil, I get `AttributeError: module 'dateutil' has no attribute 'parser'`.  I'm confused by this since it is in `dateutil.__all__`, but because of that, I'm tempted to leave it as is.

Reply:
```
>>> import dateutil
>>> dateutil.parser
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: module 'dateutil' has no attribute 'parser'
>>> import dateutil.parser
>>> dateutil.parser
<module 'dateutil.parser' from '/usr/local/lib/python3.7/site-packages/dateutil/parser/__init__.py'>
```
so you can `import dateutil.parser` and use it as `dateutil.parser` in the code.

Reply:
OK.  Done.